blueprints.yaml

```diff
@@ -20,12 +20,30 @@ form:
   validation: loose
   fields:
     enabled:
-      type: toggle
       label: PLUGIN_ADMIN.PLUGIN_STATUS
+      type: toggle
```

Comment:
Why the order change?

Reply:
Because I like to have the label first. But it's not really important for me.